piv/src/main/java/com/yubico/yubikit/piv/Padding.java

```diff
@@ -75,7 +77,9 @@ static byte[] pad(KeyType keyType, byte[] message, Signature algorithm) throws N
                 }
                 String hashAlgorithm = matcher.group(1);
                 byte[] hash;
-                if ("NONE".equals(hashAlgorithm)) {
+                if (MessageDigest.isEqual(
```

Comment:
Why are we using MessageDigest.isEqual to compare two Strings which are not digests?

Reply:
This is recommended fix for [UNSAFE_HASH_EQUAL](https://find-sec-bugs.github.io/bugs.htm#UNSAFE_HASH_EQUALS)

Reply:
I see now, you are right! This will be reverted.